sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_automation_client.py

```diff
@@ -124,339 +130,457 @@ def get_call_connection(
 
-        :return: Instance of CallConnectionClient.
+        :param call_connection_id: CallConnectionId of ongoing call.
+        :type call_connection_id: str
+        :return: CallConnectionClient
         :rtype: ~azure.communication.callautomation.CallConnectionClient
         """
         if not call_connection_id:
             raise ValueError("call_connection_id can not be None")
 
-        return CallConnectionClient(
-            call_connection_id,
-            self._call_connection_client,
-            self._call_media,
-            **kwargs
-        )
-
-    def get_call_recording(
-        self,
-        **kwargs
-    ) -> CallRecordingClient:
-        """Get CallRecordingClient object.
-        For any recording related action, use this to perform actions.
-
-        :return: Instance of CallRecordingClient.
-        :rtype: ~azure.communication.callautomation.CallRecordingClient
-        """
-        return CallRecordingClient(
-            self._call_recording_client,
-            **kwargs
-        )
+        return CallConnectionClient._from_callautomation_client( #pylint:disable=protected-access
+            callautomation_client=self._client,
+            call_connection_id=call_connection_id,
+            **kwargs)
 
+    @distributed_trace
     def create_call(
         self,
-        target: CallInvite,
-        callback_uri: str,
+        target_participant: 'CallInvite',
+        callback_url: str,
+        *,
+        operation_context: Optional[str] = None,
+        media_streaming_configuration: Optional['MediaStreamingConfiguration'] = None,
+        azure_cognitive_services_endpoint_url: Optional[str] = None,
         **kwargs
-    ) -> CreateCallResult:
+    ) -> CallConnectionProperties:
+        """Create a call connection request to a target identity.
+
+        :param target_participant: Call invitee's information.
+        :type target_participant: ~azure.communication.callautomation.CallInvite
+        :param callback_url: The call back url where callback events are sent.
+        :type callback_url: str
+        :keyword operation_context: Value that can be used to track the call and its associated events.
+        :paramtype operation_context: str
+        :keyword media_streaming_configuration: Media Streaming Configuration.
+        :paramtype media_streaming_configuration: ~azure.communication.callautomation.MediaStreamingConfiguration
+        :keyword azure_cognitive_services_endpoint_url:
+         The identifier of the Cognitive Service resource assigned to this call.
+        :paramtype azure_cognitive_services_endpoint_url: str
+        :return: CallConnectionProperties
+        :rtype: ~azure.communication.callautomation.CallConnectionProperties
+        :raises ~azure.core.exceptions.HttpResponseError:
         """
-        Create a call connection request from a source identity to a target identity.
+        if not target_participant:
```

Comment:
According to the SDK guideline, seems like we should only validate parameters in Clients; service parameters should not be validated: https://azure.github.io/azure-sdk/python_design.html#parameter-validation